src/components/Header/HeaderContact.js

```diff
@@ -46,7 +46,7 @@ const HeaderContact = ({
   return (
     <View style={[styles.container, containerStyle]}>
       {renderHeader()}
-      <Text style={[styles.text, titleStyle]}>{title}</Text>
+      <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `styles.titleText` to the Text component's style prop is a good practice for separating concerns and improving readability. This allows you to define the title text style in one place and reuse it across your application.
```diff
- <Text style={[styles.text, titleStyle]}>{title}</Text>
+ <Text style={[styles.text, styles.titleText, titleStyle]}>{title}</Text>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->